ruff.toml

```diff
@@ -1,5 +1,17 @@
 select = ["D", "E", "F", "NPY", "PD", "C4", "I"]
-ignore = ["D1", "D202", "D205", "D400", "D404", "E501", "PD002", "PD901"]
+ignore = [
+    "D1",
+    "D202",
+    "D205",
+    "D206",
```

Comment:
You added D206, D402 and E703? Might be good to place comments here about the why/what, or link to a page where we can immediately check these things.

Reply:
I've added a link to the docs